unit/solvers/smt2_incremental/smt_bit_vector_theory.cpp

```diff
@@ -174,6 +174,32 @@ TEST_CASE("SMT bit vector bitwise operators", "[core][smt2_incremental]")
   }
 }
 
+TEST_CASE("SMT bit vector comparison", "[core][smt2_incremental]")
+{
+  const smt_bit_vector_constant_termt a_valid{42, 16}, b_valid{8, 16};
+  SECTION("Valid operands")
+  {
+    const auto concat = smt_bit_vector_theoryt::compare(a_valid, b_valid);
```

Comment:
Probably best to rename `concat` here to `compare` to make the code clearer.

Reply:
Done.